views/base.njk

```diff
@@ -24,10 +24,21 @@
         <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin="">
         <link href="https://fonts.googleapis.com/css?family=Lato:700|Noto+Sans:400,700&amp;display=fallback" rel="stylesheet">
         <link rel="stylesheet" href="/dist/css/styles.css">
+        <!-- Google Tag Manager -->
+        <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+        })(window,document,'script','dataLayer','GTM-MTM7CPP');</script>
+        <!-- End Google Tag Manager -->
         <title>{% if(title) %}{{ title }} — {% endif %}{{ __('app.title') }}</title>
         {% include "_includes/ga.njk" %}
```

Comment:
Hey Khalid does this redo any of the work done in the ga.njk include? [(link to file)](https://github.com/cds-snc/c19-benefits-node/blob/master/views/_includes/ga.njk)

Reply:
No impact at all, this will just get our GTM engine fire along with GA 👍